src/naming/Naming_AST.ml

```diff
@@ -294,7 +310,13 @@ let lookup_scope_opt (s, _) env =
         *)
         | _ -> [ xs ] @ xxs @ [ !(scopes.global); !(scopes.imported) ])
   in
-  lookup s actual_scopes
+  (* actual_scopes |> List.iter (fun scope ->
```

Comment:
maybe remove the debugging code?

Reply:
Whoops

Reply:
I added another Semgrep rule to spot `$LOGGER#flash` inside comments.